conf/routes

```diff
@@ -152,6 +152,8 @@ POST    /userapi/logWebpageActivity                          @controllers.UserCo
 
 PUT    /userapi/setUserOrg/:orgId                            @controllers.UserProfileController.setUserOrg(orgId: Int)
 
+POST   /userapi/createTeam                                    @controllers.UserProfileController.createTeam
```

Comment:
So minor, but it's just not quite in line with all the other routes :sweat_smile: